rules/detekt/src/main/kotlin/com/twitter/compose/rules/detekt/ComposePreviewPublicCheck.kt

```diff
@@ -14,8 +14,6 @@ class ComposePreviewPublicCheck(config: Config) :
     TwitterDetektRule(config),
     ComposeKtVisitor by ComposePreviewPublic() {
 
-    override val autoCorrect: Boolean = true
```

Comment:
this is on purpose, Detekt is super weird with code changes so in a prior patch I removed them all (if they want autofix people can use detekt-formatting which uses ktlint under the hood) -- but left this one behind by mistake.